content/chainguard/chainguard-enforce/vulnerabilities/vuln-reports-and-attestations.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Vulnerability reports and Attestations in Chainguard Enforce"
+linktitle: "Vulnerability reports and Attestations"
+aliases:
+- /chainguard/chainguard-enforce/chainguard-enforce-kubernetes/vuln-reports-and-attestations/
+- /chainguard/chainguard-enforce/concepts/vuln-reports-and-attestations/
+type: "article"
+description: "An overview of how Chainguard Enforce handles attestations of vulnerability reports"
+date: 2023-03-19T15:56:52-07:00
+lastmod: 2023-03-19T15:56:52-07:00
+draft: false
+tags: ["ENFORCE", "COSIGN", "VULNERABILITIES", "CONCEPTUAL"]
+images: []
+menu:
+  docs:
+    parent: "concepts"
+weight: 030
+toc: true
+---
+
+In order to fully leverage the capabilities offered by Chainguard Enforce, it's important to establish a process for generating quality vulnerability reports.
+A vulnerability report lists all the discovered vulnerabilities contained in your software. It uses a [SBOM](/open-source/sbom/) to search for vulnerabilities matching the used packages in your software.
+Chainguard Enforce can capture vulnerabilities data and use it to help you find all the vulnerabilities of the packages in your ecosystem, as well as apply and enforce security policies to your runtime environments.
+To get the best results from these features, vulnerability reports should be attached to the images in your registry as signed attestations.
+
+[Cosign](/open-source/sigstore/cosign/an-introduction-to-cosign/)  — a part of the Sigstore project — supports software artifact signing, verification, and storage in an [OCI (Open Container Initiative)](/open-source/oci/) registry.
+The `cosign` command line tool offers one subcommand that you can use to attach a vulnerability report with a container image and then upload them to a registry: `cosign attest`.
+
+[Grype](https://github.com/anchore/grype)  - a vulnerability scanner for container images and filesystems.
+
+This guide outlines how to use `cosign` and `grype` to generate a cosign vulnerability attestation.
+
+## Cosign Vulnerability Attestations
+
+An *attestation* allows the end users or consumers of a software artifact (in the context of this guide, a vulnerability report) to verify — independently of the producer — that the contents of the artifact haven't been changed since it was produced. It also requires software producers to provide verifiable proof of the quality of their software.
```

Comment:
```suggestion
An *attestation* allows the end users or consumers of a software artifact (in the context of this guide, a vulnerability report) to verify that the contents of the artifact haven't been changed since it was produced. Importantly, this verification is done independently of the producer and requires software producers to provide verifiable proof of the quality of their software.
```

Maybe too long or wordy of a rewrite?